Assets/Scripts/Cards/CardDisplay.cs

```diff
@@ -0,0 +1,90 @@
+// +--------------------------------------------------------------+
+// @author - Ryan Herwig
+// @Contributers - 
+// @Last modified - September 4 2024
+// @Description - Displays the card onto an instantiated image.
+//                Also holds helper methods for an Event Trigger
+// +--------------------------------------------------------------+
+
+using UnityEngine;
+using UnityEngine.UI;
+
+public class CardDisplay : MonoBehaviour
+{
+    [SerializeField] private Card card;
```

Comment:
reminder for private variables we are using a _ before the name. so card should be _card, and etc

Reply:
If you didn't know you can hilight the variable and right click on it to rename it. please rename these three vars to comply with our style guide